packages/picasso/src/Notes/test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { render, RenderResult } from '@toptal/picasso/test-utils'
+
+import Notes from './Notes'
+
+const renderNotes = () => {
+  return render(
+    <Notes>
+      <Notes.Title>Title</Notes.Title>
+      <Notes.Subtitle>Subtitle</Notes.Subtitle>
+      <Notes.Content>Content</Notes.Content>
+    </Notes>
+  )
+}
+
+describe('Notes', () => {
+  let api: RenderResult
+
+  beforeEach(() => {
+    api = renderNotes()
+  })
+  it('default render', () => {
+    const { container } = api
+
+    expect(container).toMatchSnapshot()
+  })
+})
```

Comment:
Was extraction of `api` to the upper scope intentional?

```suggestion
describe('Notes', () => {
  it('default render', () => {
    const { container } = renderNotes()

    expect(container).toMatchSnapshot()
  })
})
```